cmd/utils.go

```diff
@@ -132,3 +132,21 @@ func lineTrunc(content string, maxLen int) string {
 	sndPart := string(runes[rlen-halfLen:])
 	return fstPart + "…" + sndPart
 }
+
+// isOlder returns true if the passed object is older than olderRef
+func isOlder(c *clientContent, olderRef int) bool {
+	objectAge := UTCNow().Sub(c.Time)
+	if objectAge < (time.Duration(olderRef) * Day) {
```

Comment:
you could just return the condition i.e. `if` is not needed.

Reply:
> you could just return the condition i.e. if is not needed.

Fixed